lib/Runtime/Base/FunctionBody.cpp

```diff
@@ -9708,7 +9706,7 @@ namespace Js
                 currentCookie = (void*)currentNativeCodegen;
 #endif
 
-                if (this->jsMethod == reinterpret_cast<Js::JavascriptMethod>(this->GetNativeAddress()))
+                if (this->jsMethod == this->GetNativeEntrypoint())
```

Comment:
The functional part of the change is the change of the function called here so that we consider the thunk if we have one, with the rest of the changes being entirely clean-up, correct? If so, then LGTM.